python/lsst/ip/diffim/dcrModel.py

```diff
@@ -319,6 +321,9 @@ def buildMatchedTemplate(self, exposure=None, order=3,
         splitSubfilters : `bool`, optional
             Calculate DCR for two evenly-spaced wavelengths in each subfilter,
             instead of at the midpoint. Default: False
+        amplifyModel : `float`, optional
+            Amplify the differences between model planes.
+            Used to speed convergence of iterative forward modeling.
```

Comment:
Is the `float` some kind of multiplier (as I assume from the default)? If so, please say so.